h/browser/chrome/lib/fetch.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Performs a network fetch for JSON data and returns a Promise
+ * for the result.
+ */
+function fetchJSON(url) {
```

Comment:
Done - Added the 'whatwg-fetch' polyfill (isomorphic-fetch is a wrapper around that, which is also Node-compatible, but we only need the client-side one)